docs/core_concepts/16_roles_and_permissions/index.mdx

```diff
@@ -0,0 +1,69 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Roles and Permissions
+
+Windmill provides a roles and permissions system that allows you to control access and manage permissions within your instance and workspaces. Different roles have different levels of access to scripts, flows, apps, and other entities in Windmill.
+
+## Roles in Windmill
+
+1. **Superadmin**: The superadmin role has the highest level of access and can manage the entire Windmill instance, including all its workspaces and invites. Superadmins have full control over the configuration and settings of the Windmill instance.
+
+2. **Admin**: Admins have the ability to manage a specific Windmill workspace and invites within that workspace. They can see and edit all scripts, flows, apps, and other entities within the workspace. Admins have the power to create, modify, and delete entities and can manage permissions within the workspace.
+
+3. **Author**: Authors can execute and view scripts, flows, and apps within a workspace. They also have the privilege of creating new scripts, flows, and apps. Authors have read and write access to the entities they create or have been [granted access to by others](../8_groups_and_folders/index.md).
```

Comment:
Authors see the full interface of windmill and contrary to the Operators, have the ability to create and edit scripts, flows and apps.